src/components/home/elements/WithdrawOrDepositToken.vue

```diff
@@ -557,9 +545,37 @@ export default {
             this.coldSignature = ''
             this.contractId = ''
             this.validContractType = true
+            this.newTokenId = ''
+            this.realFunctionName = ''
+            this.newTokenId = ''
+            this.isSplit = ''
+            this.tokenUnity = ''
+            this.tokenBalance = BigNumber(0)
         },
         hideContractError() {
-            this.validContractType = true
+            this.chain.getContractInfo(this.contractId).then(response => {
+                const SYSTEM_CONTRACT_TOKEN_ID_TEST = 'TWuKDNU1SAheHR99s1MbGZLPh1KophEmKk1eeU3mW'
```

Comment:
Move this constant to constants.js

Reply:
In fact, this SYSTEM_CONTRACT_TOKEN_ID should put in JS-SDK. Put in wallet constants.js for this commit. Then when SDK updated, import the Token ID from SDK.